python/glow/gwas/lin_reg.py

```diff
@@ -209,5 +206,37 @@ def _linear_regression_inner(genotype_pdf: pd.DataFrame, Y_state: YState,
     out_df['tvalue'] = list(np.ravel(T))
     out_df['pvalue'] = list(np.ravel(pvalues))
     out_df['phenotype'] = phenotype_names.repeat(num_genotypes).tolist()
+    if verbose_output:
+        out_df["n"] = list(np.ravel(Y_mask.T @ np.ones(X.shape)))
```

Comment:
Should we also calculate this in the earlier `if verbose_output` block to centralize these calculations?

Reply:
I think if I move some of the genotype_pdf optimization further upstream, I can do better centralize these calculations